README.md

```diff
@@ -1,5 +1,5 @@
 # soccer-stonks
-or just call it STONKSS
+or just call it RON STONKSS
 
```

Comment:
Can you also add descriptions for how to setting up venv and installing requirement.txt here, as well as we can add info on running kedro & streamlit

Reply:
Hey Ahmet! I made the changes that you requested.